packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -182,6 +190,17 @@ const Updates = decorate([
         jobs !== undefined &&
         backupNgJobs !== undefined &&
         some(jobs.concat(backupNgJobs), job => job.runId !== undefined),
+      areProxiesNeedUpdated: async (_, { proxyIds = [] }) => {
```

Comment:
`areProxiesOutdated` ?

Reply:
`areProxiesOutOfDate`.

See https://forum.wordreference.com/threads/out-of-date-outdated-no-longer-valid.3358620/#post-17023920